src/core/execute.c

```diff
@@ -4273,6 +4276,7 @@ static int exec_child(
         int secure_bits;
         _cleanup_free_ gid_t *gids_after_pam = NULL;
         int ngids_after_pam = 0;
+        uint64_t capability_ambient_set = UINT64_MAX;
```

Comment:
The initialization seems not necessary, right?

Reply:
true. fixed.